huaweicloud/utils/tags.go

```diff
@@ -38,6 +39,20 @@ func UpdateResourceTags(conn *golangsdk.ServiceClient, d *schema.ResourceData, r
 	return nil
 }
 
+//This is a help to query tags of resource, then set to state. The name must be: tags "css-cluster"
+func SetResourceTagsToState(d *schema.ResourceData, client *golangsdk.ServiceClient, resourceType string) error {
+	// set tags
+	if resourceTags, err := tags.Get(client, resourceType, d.Id()).Extract(); err == nil {
+		tagmap := TagsToMap(resourceTags.Tags)
+		if err := d.Set("tags", tagmap); err != nil {
```

Comment:
`err` has been declared above and can be used directly here.

Reply:
this is a local scope var